interface/patient_file/encounter/Tests/Service/ViewHelperTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Better to keep unit tests with the code it's actually testing instead of an isolated directory. Of course this test doesn't actually do anything at the moment